package.json

```diff
@@ -16,8 +16,8 @@
     "release": "yarn publish src",
     "lint": "yarn lint:ts && yarn lint:sol",
     "lint:fix": "yarn lint:ts:fix && yarn lint:sol:fix",
-    "lint:sol": "solhint './src/contracts/**/*.sol'",
-    "lint:sol:fix": "solhint './src/contracts/**/*.sol' --fix",
+    "lint:sol": "solhint './contracts/**/*.sol'",
+    "lint:sol:fix": "solhint './contracts/**/*.sol' --fix",
     "lint:ts": "eslint -c .eslintrc.js './**/*.ts'",
     "lint:ts:fix": "eslint -c .eslintrc.js --fix './**/*.ts'",
     "format": "prettier --write ./**/*.ts",
```

Comment:
Consider adding `forge fmt`? Perhaps format on commit hook to ensure consistency.